### PR TITLE
Implement envFile (Issue #523)

### LINKED ISSUE #523: envFile option for debugger in launch mode
This is an enhance requirement.
When configuring the debug in launch.json for launch mode. And envFile option will be helpful instead of key in all the env variables.
envFile option has been supported in c#, python, nodejs... debuggers.

### PR DESCRIPTION
**Pull request by chase**
Implemented envFile to close #523 

Uses some code from https://github.com/microsoft/vscode-js-debug to parse env files

@testforstephen

### CODE REVIEW

**Comment by ShimonBenYair** [comment]
@chase 

how can i check it works for me ?
Can you please write the steps here

**Comment by chase** [comment]
@ShimonBenYair Detailed in the test plan here:
https://github.com/microsoft/vscode-java-debug/pull/928/files#diff-0b1670dfd28e478d33ef28c79a3f975aa787121761889d142502c31e4a0641b2R437-R445

**Review by testforstephen — COMMENTED** [review]
overall looks good.

File: package.nls.zh.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "java.debugger.launch.encoding.description": "JVM的file.encoding设置。如果未指定，将使用“UTF-8”。可以在http://docs.oracle.com/javase/8/docs/technotes/guides/intl/encoding.doc.html中找到可能的值。",
     "java.debugger.launch.cwd.description": "应用程序的工作目录。默认为当前工作空间根目录。",
     "java.debugger.launch.env.description": "启动应用程序时自定义的环境变量。",
+    "java.debugger.launch.envFile.description": "包含環境變數定義之檔案的絕對路徑。",
```
[the remark this answers is not in the document]
Comment (chase) [reply package.nls.zh.json:12]: Thanks! I copied it from the extension I mentioned without giving it any thought.

**Review by testforstephen — APPROVED** [review]
LGTM. thanks for your contribution.

**Comment by troyspencer** [comment]
Thanks for working on this, I've been waiting on somebody to handle that port for a while. I built and installed from this branch and it works great, except the edge case of clicking `restart` in the debugger uses the values from the previous run instead of loading from the file like pressing `start` does. Until I realized, my iterations on environment file updates while migrating a dev setup seemed fruitless.

**Pull request merged**